js/blocks/BoxesBlocks.js

```diff
@@ -9,14 +9,18 @@
 // License along with this library; if not, write to the Free Software
 // Foundation, 51 Franklin Street, Suite 500 Boston, MA 02110-1335 USA
 
-/*
-   global
+/** 
+   @global
```

Comment:
Is this change standard? If so, it needs to be fixed in every .js file in the project.

Reply:
To maintain consistency, I'll undo that modification.